kge/config.py

```diff
@@ -423,6 +425,26 @@ def init_folder(self):
             return True
         return False
 
+    @staticmethod
+    def load_from(checkpoint_config: Config, new_config=None, folder=None) -> Config:
```

Comment:
Rename new_config to override_config to make it mode clear; add type

Reply:
done